examples/recreate_vm.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+"""
+Show how to recreate VM while maintaining VM permissions
+
+vm_name - VM to recreate
+"""
+
+import logging
+import sys
+import time
+import uuid
+
+import ovirtsdk4 as sdk
+import ovirtsdk4.types as types
+
+from helpers import jobs
+
+vm_name = 'vm'
+TIMEOUT = 100
+# Create the connection to the server:
+
+connection = sdk.Connection(
+    url='https://engine40.example.com/ovirt-engine/api',
+    username='admin@internal',
+    password='redhat123',
+    ca_file='ca.pem',
+    debug=True,
+    log=logging.getLogger(),
+)
+
+system_service = connection.system_service()
+
+vms_service = system_service.vms_service()
+
+vms = vms_service.list(search='name=%s' % vm_name)
```

Comment:
One more opportunity for f'...' string (sorry for missing it in my previous review).

Reply:
I missed this also :)

Thanks